### DISCUSSION TITLE
BirdNETpi on radxa ROCK 4C+ on debian 11 bullseye

### DISCUSSION THREAD

**Discussion by lukaz81** [mcguirepr89/BirdNET-Pi#845] [Q&A]
got it installed without errors but not able to make it record anything..

arecord shown in processes is always off
live audio is working correctly and showing histogram according to streamed audio 

unable to update via command or web interface, stuck in loop of restarting birdnet_server.service

attached are outputs from print_diagnostic_info.sh 

[print_diagnostic_info.txt](https://github.com/mcguirepr89/BirdNET-Pi/files/11204400/print_diagnostic_info.txt)

**Comment by jammbrose** [comment]
This is my experience as well.

**Comment by lukaz81** [comment]
next thing that is maybe good to mention is that with this setup  I was not able to manually record with this device to wav file, got reply that device is in use

when changed audio device back to default from previous dsnoop I was able to manualy record audio file (using arecord -D default -d 10 -f cd -t wav /tmp/test.wav), but then it looks like I lost live audio feed in birdnetPI and the arecord process is still off
